models/schedule_item.go

```diff
@@ -133,6 +133,7 @@ func getBlock(name string, StartTime time.Time) string {
 		{"YSTV", "collab"},
 		{"Nouse", "collab"},
 		{"York Politics Digest", "collab"},
+    {"Breakz", "collab"},
```

Comment:
These have rendered as spaces for me. Document uses tabs.

Reply:
@NatalieHarris2904 Make sure you have gofmt and the editorconfig setup, see the repo read me for the latter, ss for gofmt there are plugins for vscode

Reply:
Actually sorry, the editorconfig suggests spaces, will take a look at get this made clearer tomorrow night.